cmd/cluster/support/tests_helpers.go

```diff
@@ -0,0 +1,21 @@
+package support
+
+import sdk "github.com/openshift-online/ocm-sdk-go"
+
+//interface that must be met to be a SDK connection
+type SDKConnectionClient interface {
```

Comment:
This package is nice, but is it required for these tests to run? Can this refactor be done in a separate PR?

Reply:
Why is it in a file called `test_helpers` then? Can that be clarified? 

Also, as mentioned earlier, could this PR be done separately so that this refactor of how our functions work isn't lumped together with unit tests?